lib/platform.js

```diff
@@ -102,7 +96,10 @@ class TplinkSmarthomePlatform {
   addAccessory (device) {
     const deviceId = device.id;
 
-    if (this.isUnsupportedAccessory(device)) throw new Error('Missing deviceId');
+    if (deviceId == null || deviceId.length === 0) {
+      this.log.error('Missing deviceId');
```

Comment:
Continuously logs into error log -- not a huge deal but may add noise to error log with repeating "missing deviceId" every 10 seconds. Example : 
```
[2019-1-16 23:23:51] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:01] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:11] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:21] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:31] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:41] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:24:51] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:02] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:11] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:21] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:31] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:41] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:25:51] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:26:01] [TplinkSmarthome] Missing deviceId
[2019-1-16 23:26:11] [TplinkSmarthome] Missing deviceId
```

Reply:
If you're seeing that message at all that means the `filterCallback` isn't working properly. I just noticed a typo in the code and wasn't passing it properly. Can you test one more time? You shouldn't see "Missing deviceId"